decotools/fileio.py

```diff
@@ -97,7 +97,10 @@ def get_iOS_files(start_date=None, end_date=None, data_dir='/net/deco/iOSdata',
         Numpy array containing files that match specified criteria
 
     '''
-
+   
+    # Validate that data_dir exists:
+    if os.path.isdir(data_dir)==False:
+        raise IOError('data_dir, %s, cannot be located.'%(data_dir))
```

Comment:
Hey @milesjwinter, thanks for the PR! My only (super minor) comment is could you switch to using the newer `.format`-style for string formatting. That way it is consistent throughout the project.